templates/zerver/help/preview-your-message-before-sending.md

```diff
@@ -1,11 +1,13 @@
 # Previewing your message
 
-You can preview your message before you send it by clicking the **eye** (![eye](/static/images/help/eye.png)) icon.
+You can preview your message before you send it by:
 
-![Original content](/static/images/help/before-preview.png)
+1. clicking the **eye**
+(![eye](/static/images/help/eye.png)) icon at the bottom of the **Compose message box**.
 
-![Previewed](/static/images/help/during-preview.png)
+2. This is bring up the message preview:
```

Comment:
Probably, 'this *will* bring up'? I am not sure that this is a separate setup though.